### DISCUSSION TITLE
最受人民群众喜爱的直播平台评选活动

### DISCUSSION THREAD

**Discussion by luxcgo** [go-olive/olive#50] [Announcements]
**olive** 目前对我自己平时看的直播平台都已经支持了，同时所有 streamlink 支持的平台也都支持了。

但是仍然有更多的平台没有支持，其中有没有你需要的呢？
如果有的话，欢迎留下评论，高票支持的平台仍会持续加入录制支持。

**Comment by jkfujr** [comment]
tc，一个日本的直播平台
https://twitcasting.tv/

**Reply by luxcgo** [reply]
已支持

**Reply by jkfujr** [reply]
> 已支持

现版本0.3.1可以么，还有那个代理设定

**Reply by luxcgo** [reply]
0.3.2

**Reply by jkfujr** [reply]
🆗

> 0.3.2

**Reply by jkfujr** [reply]
> 0.3.2

刚测试了下录制正常，就是直播间标题的话，使用streamlink可以加上么

**Reply by jkfujr** [reply]
> 0.3.2

还有就是现版本支持全局路径的设定么

**Reply by luxcgo** [reply]
哈哈，加不上。streamlink 作者没写，除非 olivetv 那边加上。

那是啥设定

**Reply by jkfujr** [reply]
> 哈哈，加不上，streamlink 作者没写
> 
> 那是啥设定

设定好这两参数后
```
SaveDir = "E:/录播/YouTube/{{ now | date \"20060102\"}}/{{ .StreamerName }}"
OutTmpl = "{{ now | date \"20060102-150405\"}}_{{ .RoomName }}【{{ .StreamerName }}】.flv"
```
房间只需要以下配置就好了，看起来更简洁点？
```
[[Shows]]
Platform = "类型"
RoomID = "房间ID"
StreamerName = "主播名"
```

噢对了，Platform 是否也能作为配置项使用呢，比如 `{{ .Platform }}`

**Comment by yburger** [comment]
腾讯会议，一直没找到合适的录制工具，只能开虚拟机录屏
https://meeting.tencent.com/

**Reply by daxiaamo** [reply]
AfreecaTV，一个韩国的直播平台，有很多漂亮的韩国小姐姐和有名气的电竞选手。
https://www.afreecatv.com

**Comment by zjh-nuc-AIOT** [comment]
bigo live 轩子老师在

**Comment by nbqz** [comment]
cc直播支持吗？

**Comment by ChooseBetterr** [comment]
斗鱼douyu

**Comment by qzsde** [comment]
huajiao
